brain/converter/crypto.py

```diff
@@ -10,12 +10,12 @@
 import base64
 import yaml
 
-# salt_length = current_app.config.get('SALT_LENGTH')
 with open("/vagrant/hiera/settings.yaml", 'r') as stream:
     try:
         salt_length = yaml.load(stream)['crypto']['salt_length']
     except yaml.YAMLError as exc:
         print(exc)
+salt_length = current_app.config.get('SALT_LENGTH')
```

Comment:
Since this issue contains code that supersedes #2700 (because you merged a completed #2700 into #2706, then worked on #2706), you can probably replace this entire `crypto.py` with your most previous commit (i.e. ab5b6f3). associated with`crypto.py`.